lib/eb_deployer/aws_driver/beanstalk.rb

```diff
@@ -91,7 +91,8 @@ def fetch_events(app_name, env_name, params, &block)
 
       def environment_cname_prefix(app_name, env_name)
         cname = environment_cname(app_name, env_name)
-        if cname =~ /^(.+)\.elasticbeanstalk\.com/
+        region = Aws.config[:region]
+        if cname =~ /^(.+?)(\.#{region})?\.elasticbeanstalk\.com/
```

Comment:
I'm fairly certain the AWS SDK requires the region to be lower case, as I get an error when I try using uppercase in the region name, but it won't hurt to add a `.downcase` to make sure. 

I see that the automated CI build failed for ruby 1.9.3. I'm not a ruby developer can you explain why? Is there something I need to fix?